build-farm/platform-specific-configurations/linux.sh

```diff
@@ -134,7 +134,12 @@ if [ "$JAVA_FEATURE_VERSION" -gt 11 ]; then
     "$JDK_BOOT_DIR/bin/java" -version 2>&1 | sed 's/^/BOOT JDK: /'
 fi
 
-if [ -r /usr/local/gcc/bin/gcc-7.5 ]; then
+
+if [ "${VARIANT}" == "${BUILD_VARIANT_DRAGONWELL}" ] && [ "$JAVA_FEATURE_VERSION" -eq 11 ] && [ -r /usr/local/gcc9/ ]; then
+  export PATH=/usr/local/gcc9/bin:$PATH
+  [ -r  /usr/local/gcc9/ ] && export CC=/usr/local/gcc9/bin/gcc-9.3
```

Comment:
You already did `[ -r /usr/local/gcc9 ]`, so all those additional checks seem unnecessary. Or do you want to test for the existence of the individual binaries gcc/g++? If the latter, wouldn't a single condition not make more sense? Having gcc 7 and g++ 9 simultaneously enabled does not seem desirable.

Reply:
removed this check